ui/pages/swaps/view-quote/view-quote.js

```diff
@@ -672,93 +680,147 @@ export default function ViewQuote() {
     }
   }, [dispatch, viewQuotePageLoadedEvent, reviewSwapClickedTimestamp]);
 
+  const transaction = {
+    userFeeLevel: swapsUserFeeLevel || GAS_RECOMMENDATIONS.HIGH,
+    txParams: {
+      maxFeePerGas,
+      maxPriorityFeePerGas,
+      gas: maxGasLimit,
+    },
+  };
+
   return (
-    <div className="view-quote">
```

Comment:
@jpuri this diff is really hard to parse, do you mind explaining what's changed in the return block here?

Reply:
The main changes in this are:

1. Wrapping jsx in providers
```
    <GasFeeContextProvider
      editGasMode={EDIT_GAS_MODES.SWAPS}
      minimumGasLimit={usedGasLimit}
      transaction={transaction}
    >
      <TransactionModalContextProvider captureEventEnabled={false}>
```

2. Adding modals:
```
            <EditGasFeePopover />
            <AdvancedGasFeePopover />
```

3. Passing supportEIP1559V2 to FeeCard component:
```
supportsEIP1559V2={
                  EIP_1559_V2_ENABLED && networkAndAccountSupports1559
                }
```